src/pool/rpc.h

```diff
@@ -61,6 +61,9 @@
 	X(POOL_QUERY,							\
 		0, &CQF_pool_query,					\
 		ds_pool_query_handler, NULL),				\
+	X(POOL_TGT_INFO,						\
```

Comment:
On second thought, I think POOL_QUERY_TARGET could be better. Using TARGET to differentiate the TGT RPCs between server? Or POOL_QUERY_INFO?